scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -133,7 +133,7 @@
 {% endmacro %}
 
 {% macro acmg_form(institute, case, variant, ACMG_OPTIONS, selected=None) %}
-  <label class="mt-3">ACMG classification</label>
+  <label class="mt-3" data-bs-toggle="tooltip" title="Richards et al 2015 (PMID:25741868)">ACMG classification</label>
```

Comment:
We could make these clickable links, but hey, thats really just a QualityOfLife thing. Great to have the tooltip!